Tests/ML/augmentations/test_image_transforms.py

```diff
@@ -0,0 +1,97 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import random
+
+import numpy as np
+import pytest
+
+import torch
+
+from scipy.ndimage import gaussian_filter, map_coordinates
+
+from InnerEye.ML.augmentations.image_transforms import AddGaussianNoise, ElasticTransform, ExpandChannels, RandomGamma
+
+
+def test_add_gaussian_noise() -> None:
+    """
+    Tests functionality of add gaussian noise
+    """
+    np.random.seed(1)
+    tensor_img = torch.ones([1, 1, 256, 256], dtype=torch.float)
+    tensor_img[..., 100:150, 100:200] = 1 / 255.
```

Comment:
here and in the code below: There are lots of repeated constants (shapes, etc), maybe reduce some of the duplication (it's a test, so I would not be too picky, but it's really a lot of duplication)

Reply:
updated